tree/dataframe/src/RLoopManager.cxx

```diff
@@ -694,6 +694,8 @@ void RLoopManager::CleanUpTask(TTreeReader *r, unsigned int slot)
       ptr->FinalizeSlot(slot);
    for (auto &ptr : fBookedFilters)
       ptr->FinaliseSlot(slot);
+   for (auto &ptr : fBookedDefines)
```

Comment:
Should this commit be merged with https://github.com/root-project/root/commit/7c6611eabf0b85fdbc77772dddffaa5ed8ef5e26?

Reply:
yes makes sense, will do